tests/resources/telescope_positions-North-corsika.ecsv

```diff
@@ -0,0 +1,84 @@
+# %ECSV 1.0
```

Comment:
We got rid of such a file a couple of months ago in order not to duplicate the telescope list in the DB. I suggest we keep either one of them but not both

Reply:
I think the main point is to be consistent:  this is a test file for `print_array_elements`, where we convert `./tests/resources/telescope_positions-North-utm.ecsv` and test it with `telescope_positions-North-corsika.ecsv`.  We shouldn't have one file in the test directory and the other one in the DB.

I think to remember we removed it as it wasn't used in the tests, is this correct?

Reply:
I checked the DB and apparently we have already deleted these telescope position files, so it's all good.